pkg/parser/parser.go

```diff
@@ -102,10 +102,10 @@ func (p *PackageParser) Parse(ctx context.Context, reader io.ReadCloser) (*Packa
 	do := json.NewSerializerWithOptions(json.DefaultMetaFactory, p.objScheme, p.objScheme, json.SerializerOptions{Yaml: true})
 	for {
 		bytes, err := yr.Read()
-		if err != nil && err != io.EOF {
+		if err != nil && !errors.Is(err, io.EOF) {
 			return pkg, err
 		}
-		if err == io.EOF {
+		if errors.Is(err, io.EOF) {
```

Comment:
Just checked that `errors.Is` returns the same answer for `== io.EOF` by running the tests with coverage and seeing it go into this statement.